README.md

```diff
@@ -6,195 +6,252 @@
-* [Angular](https://github.com/Gbuomprisco/ngx-content-loading).
+- [React Native](https://github.com/virusvn/react-native-svg-animated-linear-gradient);
+- [Preact](https://github.com/bonitasoft/preact-content-loader);
+- Vue.js: [vue-content-loading](https://github.com/LucasLeandro1204/vue-content-loading), [vue-content-loader](https://github.com/egoist/vue-content-loader);
+- [Angular](https://github.com/Gbuomprisco/ngx-content-loading).
 
 ## Development
 
 Fork the repo then clone it
 
 `$ git clone git@github.com:YourUsername/react-content-loader.git && cd react-content-loader`
+`$ yarn`: Install the dependencies;
+`$ yarn build`: Build to production;
+`$ yarn dev`: Run the docz to see your changes;
+`$ yarn test`: Run all tests: type checking and unit tests;
+`$ yarn test:watch`: Watch unit tests;
+`$ yarn tsc`: Typescript checking;
+`$ yarn tsc:watch`: Typescript checking with watching;
 
-Install the dependencies
+## License
 
-`$ yarn`
+[MIT](https://github.com/danilowoz/react-content-loader/blob/master/LICENSE)
 
-Run the docz to see your changes
+## Known Issues
 
-`$ yarn dev`
+##### **Alpha is not working: Safari / iOS**
 
-## License
+When using `rgba` as a `primaryColor` or `secondaryColor` value, [Safari does not respect the alpha channel](https://github.com/w3c/svgwg/issues/180), meaning that the color will be opaque. To prevent this, instead of using an `rgba` value for `primaryColor`/`secondaryColor`, use the `rgb` equivalent and move the alpha channel value to the `primaryOpacity`/`secondaryOpacity` props.
 
-[MIT](https://github.com/danilowoz/react-content-loader/blob/master/LICENSE)
+```jsx
+{/* Opaque color in Safari and iOS */}
+<ContentLoader
+  primaryColor="rgba(0,0,0,0.06)"
+  secondaryColor="rgba(0,0,0,0.12)">
 
-## Known Issues
+```
 
-* **Safari / iOS**
-      
-  When using `rgba` as a `primaryColor` or `secondaryColor` value, [Safari does not respect the alpha channel](https://github.com/w3c/svgwg/issues/180), meaning that the color will be opaque. To prevent this, instead of using an `rgba` value for `primaryColor`/`secondaryColor`, use the `rgb` equivalent and move the alpha channel value to the `primaryOpacity`/`secondaryOpacity` props.
-  
-  ```jsx
-  {/* Opaque color in Safari and iOS */}
-  <ContentLoader 
-    primaryColor="rgba(0,0,0,0.06)" 
-    secondaryColor="rgba(0,0,0,0.12)">
-  
-
-  {/* Semi-transparent color in Safari and iOS */}
-  <ContentLoader 
+```
+{/_ Semi-transparent color in Safari and iOS _/}
+<ContentLoader
     primaryColor="rgb(0,0,0)"
     secondaryColor="rgb(0,0,0)"
     primaryOpacity={0.06}
     secondaryOpacity={0.12}>
-  ```
+
+```
+
+##### **Black box in Safari / iOS (again)**
+
+Using base tag on a page that contains SVG elements fails to render, it looks like a black box. Just remove the **base-href** tag from `<head />` and issue solved. 
```

Comment:
Using base tag on a page that contains SVG elements fails to render and it looks like a black box. Just remove the **base-href** tag from the `<head />` and issue solved.